devicetypes/smartthings/zigbee-smoke-sensor.src/zigbee-smoke-sensor.groovy

```diff
@@ -135,11 +135,11 @@ def ping() {
 }
 def configure() {
 	log.debug "configure"
-	sendEvent(name: "checkInterval", value:6 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
+	sendEvent(name: "checkInterval", value:20 * 60 + 2 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
```

Comment:
ZoneStatus is configured to report every 180 seconds so it shouldn't take 20 minutes to determine the device is offline. I suggest setting `checkInterval` to 12 minutes like most other devices, even though this one is configured to report a little more often.